File: works/10817.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;
import static java.util.Collections.sort;

public class Main
{
    public static void main(String[] args) throws IOException
    {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] _s = br.readLine().split(" ");
        ArrayList<Integer> v = new ArrayList<Integer>();
        v.add(Integer.parseInt(_s[0]));
        v.add(Integer.parseInt(_s[1]));
        v.add(Integer.parseInt(_s[2]));
        sort(v);
        System.out.print(v.get(1));
    }
}

File: works/10817.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.IO;
using static System.Console;

namespace ConsoleApp
{
    class Program
    {


        static void Main(string[] args)
        {
            StreamReader sr = new StreamReader(Console.OpenStandardInput());
            string[] _s = sr.ReadLine().Split(' ');
            ArrayList v = new ArrayList();
            v.Add(int.Parse(_s[0]));
            v.Add(int.Parse(_s[1]));
            v.Add(int.Parse(_s[2]));
            v.Sort();
            Write(v[1]);
        }

    }
}
